src/popover.scss

```diff
@@ -226,6 +235,12 @@ $block: #{$fd-namespace}-popover;
       }
     }
 
+    &--input-message-group {
+      box-shadow: none;
+      border: none;
+      margin-top: -0.25rem !important;
```

Comment:
is there a way to do this without using important?

Reply:
no

Reply:
I'm also no fan of using `!important`. It's in modifier, so it shouldn't be a problem, I saw that bootstrap 3/4 uses `!imrpotant` in some places. After removing important, it works actually
Without:
![image](https://user-images.githubusercontent.com/26483208/74434711-3d931680-4e63-11ea-9322-87086154c4a7.png)

With:
![image](https://user-images.githubusercontent.com/26483208/74434736-4a176f00-4e63-11ea-881c-a2ac2672942c.png)

Reply:
How do you test popper.js inside fundamental-styles?